src/discord_mcp/core/plugins/__init__.py

```diff
@@ -0,0 +1,8 @@
+from .manager import DiscordMCPPluginManager, PromptManifest, ResourceManifest, ToolManifest
+
+__all__: tuple[str, ...] = (
+    "DiscordMCPPluginManager",
+    "ToolManifest",
+    "ResourceManifest",
+    "PromptManifest",
+)
```

Comment:
```suggestion
from .manager import DiscordMCPPluginManager

__all__: tuple[str, ...] = (
    "DiscordMCPPluginManager",
)
```
Manifests aren't actually defined there as I have said in another comment, sorry 🙏